apps/ledger-live-desktop/src/renderer/screens/account/index.jsx

```diff
@@ -93,6 +93,18 @@ const AccountPage = ({
 
   const isCompoundEnabled = useCompoundAccountEnabled(account, parentAccount);
 
+  const [banner, setBanner] = useState({});
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const bannerProps = getBannerProps(account, { t, dispatch });
+      setBanner(bannerProps);
+    };
+    fetchData();
```

Comment:
👋  you still need to do a `if (account)` i think, account could become undefined when deleted 🤔